src/nvim/terminal.c

```diff
@@ -217,10 +214,14 @@ Terminal *terminal_open(TerminalOptions opts)
   vterm_set_utf8(rv->vt, 1);
   // Setup state
   VTermState *state = vterm_obtain_state(rv->vt);
+  // Only relevant in terminal true color mode
+  if (true_color) {
+    vterm_state_set_bold_highbright(state, true);
```

Comment:
@DarkDefender That makes sense, putting the option `g:terminal_bold_highbright` after the true color check seems reasonable. I wouldn't be able to fake highlight off because terminal application themes I use use all the 16 base colours.